app/Shop/Couriers/CourierRepository.php

```diff
@@ -95,11 +100,23 @@ public function deleteCourier()
         return $this->delete();
     }
 
-    public function getAvailableCouriers(){
-
+    public function getAvailableCouriers(object $cartItemWeights){
 //        TODO dorobit validaciu rozmerov!
 
-        return Courier::with('paymentMethods')->get();
+        $totalWeight = $cartItemWeights->reduce(function ($total, $item) {
+            return $total + $item;
+        });
+
+        $couriersLowerLimit= Courier::all()
```

Comment:
V Laraveli su dve veci, jedna vec su Kolekcie a druha vec je Databazove query. Ked zavolas all() tak sa ti spravi v SQL SELECT * FROM couriers a s tym vysledkom sa pracuje. Najlepsie ale je ak co najviac toho spravis este v tej DB query, cize priamo v nej sa da selectnut vsetko s podmienkou (where where) OR where

Reply:
No jo a este to musis vratis aj s tymi payment metodami, dala si prec to with